index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8"/>
-    <title>Linux.org.ph</title>
+    <title><title>Philippine Linux Users' Group - plug.linux.org.ph</title>
```

Comment:
Tag got doubled, easy fix:

```suggestion
    <title>Philippine Linux Users' Group - plug.linux.org.ph</title>
```